spec/lib/jobs/create-default-pollers-spec.js

```diff
@@ -235,16 +235,19 @@ describe("Job.Pollers.CreateDefault", function () {
         .then(function() {
             expect(waterline.obms.findByNode).to.have.been.callCount(4);
             expect(waterline.workitems.findOrCreate).to.have.been.callCount(4);
-            _.forEach(nodeIds, function(nodeId){
-                expect(waterline.obms.findByNode).to.have.been.calledWith(
-                    nodeId, 'ucs-obm-service', true
-                );
-                expect(waterline.workitems.findOrCreate).to.have.been.calledWith(
-                    { node: nodeId, 'config.command': pollers[0].config.command }, pollers[0]
-                );
-                expect(waterline.workitems.findOrCreate).to.have.been.calledWith(
-                    { node: nodeId, 'config.command': pollers[1].config.command }, pollers[1]
-                );
+            expect(waterline.obms.findByNode).to.have.been.calledWith(
+                nodeIds[0], 'ucs-obm-service', true
+            );
+            expect(waterline.obms.findByNode).to.have.been.calledWith(
+                nodeIds[1], 'ucs-obm-service', true
+            );
+
+            var arrayLength = pollers.length * nodeIds.length;
+            _.each(Array.from({length: arrayLength}, function(v, i){ return i; }),
```

Comment:
_.forEach has the same functionality and is more popular than _.each.

Reply:
fixed.